src/c/native_ledger_client.cc

```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2018 Brave Software. Distributed under the MPL2 license.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "src/c/native_ledger_client.h"
+
+#include "bat-native-ledger/include/bat/ledger/ledger.h"
```

Comment:
this should be "bat/ledger/ledger.h" so I think maybe there's an issue with your include paths in gyp